src/cryptoadvance/specter/templates/includes/tx-table.html

```diff
@@ -939,7 +939,14 @@ <h1>{{ _("Export transactions to CSV") }}</h1>
                 } else {
                     pending_psbts = []
                 }
-                for (let txRow of this.shadowRoot.querySelectorAll('tx-row')) { 
+                for (let txRow of this.shadowRoot.querySelectorAll('tx-row')) {
```

Comment:
I've added a check here to ensure that the rbf buttons are hidden on the UTXO overview. See comment for why this is not strictly necessary but still more future proof in case the UTXO api endpoint return values change.